### DISCUSSION TITLE
Cannot find documentation on template variables... (showRssFeed, showToggleTheme, etc)

### DISCUSSION THREAD

**Discussion by BallisticPain** [arthelokyo/astrowind#527] [Q&A]
I'm brand new to Astro. I'm not new to development. I found your template and it is what made me decide to check out Astro for these projects I'm working on. I love the work and effort you have put into this. I look forward to AstroWind v2. I am happy to create a PR to add some documentation about using some of these more advanced boolean flags. I tried to go through the code and figure out how to use/pass them through. I'm sure if I understood more about Astro, I probably would know the answer, but since I don't, I would love to learn and then help this project by getting it into some documentation!

So... I'll track through what I've found. What I would really like is to remove the RSS button next to the CtA button in the top right of the `index.astro` page. I figured out that the Header.astro is what contains `showRssFeed` variable. This is passed in through the `PageLayout.astro`. I haven't figured out where that page gets it's variables. There is another two variables getting passed into the `<Header>` tag ... `isSticky` and `showToggleTheme`. I've also not been able to find those. I assume this means they are Astro variables that I would need to put into a configuration somewhere.

If someone can direct me where I should put those, I would greatly appreciate it. I would be happy to make a PR and get some examples into the documentation. I believe since AstroWind is a popular template, you will get a lot of Astro newbies. The more examples we can get into your documentation the faster we'll all get up to speed.

One other small thing I noticed. I couldn't figure out how to "download" the template to use it. It was much easier once I put the Astro docs to use... I used `npm create astro@latest -- --template onwidget/astrowind` which made it quite easy. I had to try a couple of times to get that working properly, but that line got me going immediately.

**Comment by BallisticPain** [comment]
Just putting some other ideas in here that I feel I should document when I submit my PR...

`flat-color-icons` ... it took me a while to find how to include additional icons from this set. I was able to work my way backwards, but some links in the documentation would have helped me go faster. I tried searching in this repository, but it wouldn't pull up anything other than the files that were using the icons. Not the configuration file `astro.config.mjs` where I needed to input those additional icons I wanted.

Thanks again for such a great template to work with! It has been a pleasure even with these hiccups. I'm used to having to dig... :)

**Comment by SamKr** [comment]
Hi @BallisticPain,

Brand new to Astro as well, and came here with the same question. I found the const props containing `showRssFeed` in `Header.astro`, but the const set there aren't used. I can't for the life of me figure out where they do get set. 

Did you find the answer yet, or, can someone point us in the right direction?

**Comment by djagoo** [comment]
Was anyone able to find an answer to that question?

**Comment by SamKr** [comment]
I haven't yet

**Comment by nicovtx** [comment]
In Header.astro this code is displaying the RSS button, just put it in comment 
{
        showRssFeed && (
              <a
                class="text-muted dark:text-gray-400 hover:bg-gray-100 dark:hover:bg-gray-700 focus:outline-none focus:ring-4 focus:ring-gray-200 dark:focus:ring-gray-700 rounded-lg text-sm p-2.5 inline-flex items-center"
                aria-label="RSS Feed"
                href={getAsset('/rss.xml')}
              >
                <Icon name="tabler:rss" class="w-5 h-5" />
              </a>
            )
}

**Reply by djagoo** [reply]
Thats the workaround i currently use. Thanks.

**Comment by spjohn85** [comment]
I removed them from the src/layouts/PageLayout.astro and its removed in the Headers,

```
  <slot name="header">
    <Header {...headerData} isSticky showToggleTheme />
  </slot>
```
